tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -123,30 +125,37 @@ export default function CalendarScreen ({ route, navigation }) {
                         }
                     }
                     setMarked(markedState => ({...markedState, ...newMarkedData}));
-                } 
+                }
             }
         }
 
         fetchYearData()
-    }, [yearInView]) 
+    }, [yearInView])
 
     useFocusEffect(
         useCallback(() => {
+
+            // set newly marked calendar dates with changed symptoms
             let newMarkedData = route.params?.inputData
             if (newMarkedData) {
                 setMarked(markedState => ({...markedState, ...newMarkedData}));
             }
 
+            // show tutorial overlay if coming from Confirmation screen, else ignore
+            AsyncStorage.getItem('showTutorial')
```

Comment:
Can you turn this into a function to put in a /service file? It would be easier because in the .catch() of all service files should have the error popup when error is caught

Reply:
Done!